autoload/ctrlp/mrufiles.vim

```diff
@@ -56,16 +57,22 @@ fu! s:reformat(mrufs, ...)
 endf
 
 fu! s:record(bufnr)
-	if s:exclnomod && &l:modifiable | retu | en
-  if s:exclnomod && !&l:modifiable | retu | en
```

Comment:
True, it does always exit early. 

Turns out my check
```vim
		 !empty(getbufvar('^' . abs_fn . '$', '&bt')) 
```
suffices to detect help buffers. So I think the no-mod check is mostly superfluous, as it was mainly aimed at skipping helpfiles.